lib/manager/gitlabci-include/extract.ts

```diff
@@ -35,15 +35,19 @@ export function extractPackageFile(
     const doc: any = load(replaceReferenceTags(content), {
       json: true,
     });
+    let includes;
```

Comment:
```suggestion
    if (!doc?.include) {
      return null;
    }
```

early exit when no include